lib/cog_api/fake/roles.ex

```diff
@@ -75,4 +75,11 @@ defmodule CogApi.Fake.Roles do
   defp permissions_match?(%Permission{} = permission, %Permission{} = match) do
     permission.namespace == match.namespace && permission.name == match.name
   end
+
+  defp prepare_role(role) do
```

Comment:
I think I'm missing something. Why isn't this solved already by adding `groups` to the `associations` keyword list?